whatshap/vcf.py

```diff
@@ -964,7 +987,16 @@ def write(self, chromosome: str, sample_superreads, sample_components):
                         is_het = not genotypes[pos].is_homozygous()
 
                     if pos in components and pos in phases and is_het:
-                        self._set_phasing_tags(call, components[pos], phases[pos])
+                        self._set_phasing_tags(
+                            call,
+                            components[pos],
+                            phases[pos],
+                            haploid_components[pos]
+                            if haploid_components
+                            and pos in haploid_components
+                            and len(haploid_components[pos]) == self.ploidy
+                            else None,
```

Comment:
Embedding the conditional expression on multiple lines looks a bit weird and is a bit hard to read. Can you pull this out as a regular `if` that sets a temporary variable (`haploid_component` or so)?

Reply:
I changed the statement and also rebased with the current master.